MoltenVK/MoltenVK/Layers/MVKExtensions.def

```diff
@@ -58,6 +58,7 @@ MVK_EXTENSION(KHR_external_memory,                 KHR_EXTERNAL_MEMORY,
 MVK_EXTENSION(KHR_external_memory_capabilities,    KHR_EXTERNAL_MEMORY_CAPABILITIES,     INSTANCE, 10.11,  8.0)
 MVK_EXTENSION(KHR_external_semaphore,              KHR_EXTERNAL_SEMAPHORE,               DEVICE,   10.11,  8.0)
 MVK_EXTENSION(KHR_external_semaphore_capabilities, KHR_EXTERNAL_SEMAPHORE_CAPABILITIES,  INSTANCE, 10.11,  8.0)
+MVK_EXTENSION(KHR_fragment_shader_barycentric,     KHR_FRAGMENT_SHADER_BARYCENTRIC,      DEVICE,   10.15, 14.0)
```

Comment:
While we are at it, is there any reason we should not also advertise `VK_NV_fragment_shader_barycentric`, for simple legacy support?

It looks like the struct queries above would be sufficient, and it would just require adding an additional entry here.

Reply:
Please also add `VK_KHR_fragment_shader_barycentric` (and likely `VK_NV_fragment_shader_barycentric`) to the corresponding list in `Docs/MoltenVK_Runtime_UserGuide.md`.